gstd/src/msg.rs

```diff
@@ -92,8 +86,7 @@ pub fn send_with_value<E: Encode>(
     gas_limit: u64,
     value: u128,
 ) -> MessageId {
-    let bytes = payload.encode();
-    gcore::msg::send_with_value(program, &bytes, gas_limit, value)
+    send_bytes_with_value(program, &payload.encode(), gas_limit, value)
```

Comment:
Should we rename it to `send_bytes`? 
```suggestion
    send_bytes(program, &payload.encode(), gas_limit, value)
```

Reply:
We already have `send_bytes()` function, which doesn't get value as an argument. These **_with_value** functions will disappear in further api versions.